docs/research-software/openfoam/openfoam.md

```diff
@@ -56,6 +56,10 @@ need to be added to any job/batch submission scripts you want to use to
 run OpenFOAM. Note that all the centrally installed versions of OpenFOAM
 are compiled under `PrgEnv-gnu`.
 
+Note there are no default versions specificied. It is recommended to
```

Comment:
I'm not sure what you mean here -- if not specified, the default version is the last one to be compiled (in this case the .org v8 version).

Reply:
"module versions" may be more clear.